### ISSUE TITLE
Missing form-data dependency

### ISSUE THREAD

**Issue by trevorharwell** [ngrok/ngrok-api-typescript#9]
Your client source (https://github.com/ngrok/ngrok-api-typescript/blob/main/src/client.ts#L4) depends on form-data and yet there doesn't seem to be any explicit dependency on it (https://github.com/ngrok/ngrok-api-typescript/blob/main/package.json#L20-L23)

I got an error immediately upon trying to use this library saying that form-data did not exist.

Am I crazy that this library is currently broken?  I did see form-data in package-lock so maybe its a difference of npm versions (I am on npm 7.24.2)

**Comment by russorat** [comment]
Hi @trevorharwell, thanks for opening this. I am looking into this today.

**Comment by russorat** [comment]
I've run through and identified a few issues with this client library, including the ones you've mentioned. I'm working with the team to see how we can address them in the code that generates this client.

**Comment by ruscon** [comment]
bug still exists

**Comment by nijikokun** [comment]
@ruscon correct, looks like we need to do a new release. re-opening and pinging the team.
